common/common.cpp

```diff
@@ -671,7 +671,16 @@ bool gpt_params_parse_ex(int argc, char ** argv, gpt_params & params) {
         } else if (arg == "--no-mmap") {
             params.use_mmap = false;
         } else if (arg == "--numa") {
-            params.numa = true;
+            if (++i >= argc) {
+                invalid_param = true;
+                break;
+            } else {
```

Comment:
Another nitpick: This `else` is unnecessary.

Reply:
Good catch. I've made the change, completed local tests and committed